postman/hotrod.postman_collection.json

```diff
@@ -0,0 +1,309 @@
+						"packages": {}
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Accept",
+						"value": "*/*"
+					},
+					{
+						"key": "Accept-Language",
+						"value": "en-US,en;q=0.9"
+					},
+					{
+						"key": "Cache-Control",
+						"value": "no-cache"
+					},
+					{
+						"key": "Connection",
+						"value": "keep-alive"
+					},
+					{
+						"key": "Content-Type",
+						"value": "application/json"
+					},
+					{
+						"key": "Origin",
+						"value": "http://frontend.hotrod-istio.svc:8080"
+					},
+					{
+						"key": "Pragma",
+						"value": "no-cache"
+					},
+					{
+						"key": "Referer",
+						"value": "http://frontend.hotrod-istio.svc:8080/"
+					},
+					{
+						"key": "User-Agent",
+						"value": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36"
+					},
+					{
+						"key": "baggage",
+						"value": "session=1490, request=1"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\"sessionID\":1490,\"requestID\":3,\"pickupLocationID\":123,\"dropoffLocationID\":1}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "http://frontend.hotrod-istio.svc:8080/dispatch",
+					"protocol": "http",
+					"host": [
+						"frontend",
+						"hotrod-istio",
+						"svc"
+					],
+					"port": "8080",
+					"path": [
+						"dispatch"
+					]
+				}
+			},
+			"response": [
+				{
+					"name": "When dropoff location doesn't exists",
+					"originalRequest": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Accept",
+								"value": "*/*"
+							},
+							{
+								"key": "Accept-Language",
+								"value": "en-US,en;q=0.9"
+							},
+							{
+								"key": "Cache-Control",
+								"value": "no-cache"
+							},
+							{
+								"key": "Connection",
+								"value": "keep-alive"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							},
+							{
+								"key": "Origin",
+								"value": "http://frontend.hotrod-istio.svc:8080"
+							},
+							{
+								"key": "Pragma",
+								"value": "no-cache"
+							},
+							{
+								"key": "Referer",
+								"value": "http://frontend.hotrod-istio.svc:8080/"
+							},
+							{
+								"key": "User-Agent",
+								"value": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36"
+							},
+							{
+								"key": "baggage",
+								"value": "session=1490, request=1"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"sessionID\":1490,\"requestID\":3,\"pickupLocationID\":123,\"dropoffLocationID\":1123}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "http://frontend.hotrod-istio.svc:8080/dispatch",
```

Comment:
It appears to be hardcoding this variable - can you make this var be derived from the @{namespace} similar to what you did with playwright?